code/modules/clothing/gloves/gloves.dm

```diff
@@ -0,0 +1,31 @@
+/obj/item/clothing/gloves
+	name = "gloves"
+	gender = PLURAL //Carn: for grammarically correct text-parsing
+	w_class = WEIGHT_CLASS_SMALL
+	icon = 'icons/obj/clothing/gloves.dmi'
+	siemens_coefficient = 0.5
+	body_parts_covered = HANDS
+	slot_flags = SLOT_GLOVES
+	attack_verb = list("challenged")
+	var/transfer_prints = FALSE
+	strip_delay = 20
+	equip_delay_other = 40
+
+
+/obj/item/clothing/gloves/worn_overlays(isinhands = FALSE)
+	. = list()
+	if(!isinhands)
+		if(damaged_clothes)
+			. += mutable_appearance('icons/effects/item_damage.dmi', "damagedgloves")
+		if(blood_DNA)
+			. += mutable_appearance('icons/effects/blood.dmi', "bloodyhands")

```

Comment:
Does this proc really need to get copy pasted per subtype

Reply:
no but they will need vars for those overlays

Reply:
It was a rhetorical question